### DISCUSSION TITLE
PDF Header Footer removal

### DISCUSSION THREAD

**Discussion by wilsonlimaneto** [Unstructured-IO/unstructured#2006] [Q&A]
Hey I read thru the docs, but couldn´t find a way of removing PDFs headers/footers. Did I miss anything? Thanks

**Comment by jpmin7** [comment]
As far as I understand it, you partition your pdf in hi_res and get back a list of unstructured elements. Such as narrativeText and Header and Footer. You can then use the chunking functions to chunk your elements while excluding header and footers elements. Hope that helps.

**Reply by jpmin7** [reply]
https://unstructured-io.github.io/unstructured/introduction/overview.html#id1
